# --- SDE-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
#
# Filename: package/.../bcron/initgroups.patch
# Copyright (C) 2010 The OpenSDE Project
#
# More information can be found in the files COPYING and README.
#
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- SDE-COPYRIGHT-NOTE-END ---

--- ./bcron-exec.c.orig	2010-11-03 14:24:21.000000000 +0100
+++ ./bcron-exec.c	2010-11-03 14:24:33.000000000 +0100
@@ -111,7 +111,7 @@
   dup2(fdout, 1);
   dup2(fdout, 2);
   close(fdout);
-  if (0 && initgroups(pw->pw_name, pw->pw_gid) != 0)
+  if (initgroups(pw->pw_name, pw->pw_gid) != 0)
     die1sys(111, "Could not initgroups");
   if (setgid(pw->pw_gid) != 0)
     die1sys(111, "Could not setgid");
